justice/datasets/plasticc_data.py

```diff
@@ -133,13 +134,64 @@ def _sqlite_get_lc(cls, conn, dataset, obj_id):
         lc.meta = dict(zip(cls.metadata_keys, meta_row))
         return lc
 
+    @classmethod
+    def _bcolz_get_lcs(cls, bcolz_table, meta_table, obj_ids):
+        query_parts = ['(object_id == {})'.format(o) for o in obj_ids]
+        query = ' | '.join(query_parts)
+        bcolz_map = bcolz_table.where(query)
+        # TODO: make this more efficient
+        all_raw_bands = {}
+        for o in obj_ids:
+            all_raw_bands[o] = {}
+            for passband, _ in enumerate(cls.expected_bands):
+                # ('time', 'flux', 'flux_err', 'detected')
+                all_raw_bands[o][passband] = ([], [], [], [])
+        for row in bcolz_map:
+            o = row[0]
```

Comment:
nit. except for very standard notation (e.g. indices 'for i in ...'), please use larger names, e.g. `obj_id` or `object_id`

also as in-person, consider using dataframe methods or at least `bcolz_table.names` to get column names & make this more robust if we add or remove columns.

maybe https://github.com/Blosc/bcolz/blob/master/bench/pandas-todataframe.py is helpful